.travis.yml

```diff
@@ -1,60 +1,68 @@
 language: android
-dist: precise
+dist: trusty
```

Comment:
In my experience Android never works on Travis's "trusty" distribution.

Reply:
it is strange, but it works with `trusty` now